docker/build.sh

```diff
@@ -22,6 +22,7 @@ MVN_VERSION=$(xmllint --xpath "//*[local-name()='project']/*[local-name()='versi
 echo "Building with Maven"
 mvn clean
 mvn -T 2C formatter:format impsort:sort && mvn xml-format:xml-format
+mvn -T 2C compile  -P-use-sonatype-snapshots package -DskipTests -Dmaven.javadoc.skip=true -Dformatter.skip=true -Dimpsort.skip=true -Dxml-format.skip=true  -Djapicmp.skip -Denforcer.skip=true -Dbuildnumber.plugin.phase=none -Ddefault-jar.phase=none -Danimal.sniffer.skip=true
```

Comment:
This improves the build performance. It was taking 7 minutes to do a clean build, and that was very annoying when I was testing my actual changes, now it takes 2 1/2 minutes.